src/Generator/TraceId/RamseyUuid4Generator.php

```diff
@@ -10,6 +10,7 @@
 
 /**
  * Uses `ramsey/uuid` to generator v4 UUIDs for request ids.
+ * @internal
```

Comment:
@bram123  Can't make these internal. You need to reference these classes in the configuration for $config->traceId()->generatorService(RamseyUuid4Generator::class);